src/nuklear_internal.h

```diff
@@ -305,14 +308,16 @@ enum nk_property_filter {
     NK_FILTER_FLOAT
 };
 enum nk_property_kind {
-    NK_PROPERTY_INT,
-    NK_PROPERTY_FLOAT,
-    NK_PROPERTY_DOUBLE
+	NK_PROPERTY_INT,
```

Comment:
Please don't change code style if possible (see our recommended [review guidelines](https://github.com/Immediate-Mode-UI/Nuklear#reviewers-guide ) which our reviewers of PRs are encouraged to follow).

Reply:
Ok I will fix that.